problems/arrayMethods.js

```diff
@@ -14,7 +20,17 @@ const sumArray = () => {};
  * Must use forEach
  */
 
-const zeroCount = () => {};
+const zeroCount = (nums) => {
+  let count = 0;
+  nums.forEach((el) => {
+    if(el === 0){
+      count += 1;
+ //     console.log(count)
+    }
+    })
```

Comment:
Make sure you format your code right, you can use a vs code extension like [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to auto format your code when you save a file